chain/types/ethtypes/eth_types.go

```diff
@@ -556,7 +567,7 @@ type EthLog struct {
 	// The index corresponds to the sequence of messages produced by ChainGetParentMessages
 	TransactionIndex EthUint64 `json:"transactionIndex"`
 
-	// TransactionHash is the cid of the message that produced the event log.
+	// TransactionHash is the hash of the RLP message that produced the event log.
 	TransactionHash EthHash `json:"transactionHash"`
```

Comment:
Eventually we might want to return events of built-in actors transactions too, but that gets a bit trickier anyway, so this is fine.